content/en/containers/amazon_ecs/_index.md

```diff
@@ -261,7 +261,49 @@ If you already have a task definition, update your file to include the following
    "family": "datadog-agent-task"
  }
  ```
+#### Network Path
 
+<div class="alert alert-info">Network Path for Datadog Network Performance Monitoring is in private beta. Reach out to your Datadog representative to sign up.</div>
```

Comment:
```suggestion
<div class="alert alert-info">Network Path for Datadog Network Performance Monitoring is in Preview. Reach out to your Datadog representative to sign up.</div>
```
Should this be "in Preview" now?

Reply:
Hi @brett0000FF not yet for this one. The app still says "Alpha" as well as the main Network Path docs still say "private beta". Targeting Limited Availability for re:invent.. Let me check.